.github/workflows/create-release.yml

```diff
@@ -1,279 +1,539 @@
+      ubi_images_need_update: ${{ steps.compare_previous_and_current_sha256_hash_codes.outputs.ubi_images_need_update }}
+      ubi_image_previous_hash_codes: ${{ steps.compare_previous_and_current_sha256_hash_codes.outputs.ubi_image_previous_hash_codes }}
+
     steps:
-    - name: Run if stack files changed
-      run: |
-        echo "stack files have changed"
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Find and Download Previous build sha256 image hash code
+        id: fetch_previous_build_sha256_image_hash_code
+        uses: paketo-buildpacks/github-config/actions/release/find-and-download-asset@main
+        with:
+          asset_pattern: "${{ env.UBI_BUILD_SHA256_FILENAME }}"
+          search_depth: 1
+          repo: ${{ github.repository }}
+          output_path: "/github/workspace/${{ env.UBI_BUILD_SHA256_FILENAME }}"
+          token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
+
+      - name: Find and Download Previous run sha256 image hash code
+        id: fetch_previous_run_sha256_image_hash_code
+        uses: paketo-buildpacks/github-config/actions/release/find-and-download-asset@main
+        with:
+          asset_pattern: "${{ env.UBI_RUN_SHA256_FILENAME }}"
+          search_depth: 1
+          repo: ${{ github.repository }}
+          output_path: "/github/workspace/${{ env.UBI_RUN_SHA256_FILENAME }}"
+          token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
+
+      - name: Find and Download Previous run Node.js 16 sha256 image hash code
+        id: fetch_previous_run_nodejs_16_sha256_image_hash_code
+        uses: paketo-buildpacks/github-config/actions/release/find-and-download-asset@main
+        with:
+          asset_pattern: "${{ env.UBI_RUN_NODEJS_16_SHA256_FILENAME }}"
+          search_depth: 1
+          repo: ${{ github.repository }}
+          output_path: "/github/workspace/${{ env.UBI_RUN_NODEJS_16_SHA256_FILENAME }}"
+          token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
+
+      - name: Find and Download Previous run Node.js 18 sha256 image hash code
+        id: fetch_previous_run_nodejs_18_sha256_image_hash_code
+        uses: paketo-buildpacks/github-config/actions/release/find-and-download-asset@main
+        with:
+          asset_pattern: "${{ env.UBI_RUN_NODEJS_18_SHA256_FILENAME }}"
+          search_depth: 1
+          repo: ${{ github.repository }}
+          output_path: "/github/workspace/${{ env.UBI_RUN_NODEJS_18_SHA256_FILENAME }}"
+          token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
+
+      - name: Compare previous and current sha256 hash codes
+        id: compare_previous_and_current_sha256_hash_codes
+        run: |
+
+          ubi_images_need_update=()
+          ubi_image_previous_hash_codes=()
+
+          # Build Image
+          docker pull registry.access.redhat.com/ubi8/ubi-minimal
+          current_build_sha256_image_hash_code=$(docker image inspect registry.access.redhat.com/ubi8/ubi-minimal --format '{{json .RepoDigests}}' | jq -r '.[0]')
```

Comment:
We might be able to get this information without pulling the image via `skopeo` (if running on a Linux machine):

```
 skopeo inspect --format "{{.Digest}}" docker://registry.access.redhat.com/ubi8/ubi-minimal
```

If on a Mac, you might need to run with `--raw` or something.

Reply:
I got the same SHA256 doing the `docker pull`/`docker inspect` mechanism as when I did the `skopeo` mechanism

Reply:
```
skopeo inspect --format "{{.Digest}}" docker://registry.access.redhat.com/ubi8/ubi-minimal
sha256:ca2d09012e43e133f7703cbfc9ed9a6b5e5485ca7b3d52e9b3b287a0780b1bb3
```

Reply:
```
docker image inspect registry.access.redhat.com/ubi8/ubi-minimal --format '{{json .RepoDigests}}' | jq -r '.[0]'
registry.access.redhat.com/ubi8/ubi-minimal@sha256:ca2d09012e43e133f7703cbfc9ed9a6b5e5485ca7b3d52e9b3b287a0780b1bb3
```

Reply:
Added the improvement here https://github.com/paketo-community/ubi-base-stack/pull/7/commits/3e30951f8f4f19372bda9e0af1c081ed6f9a874f 
Thanks for the suggestion, really helpful!